ironmon_tracker/data/MoveData.lua

```diff
@@ -352,6 +439,20 @@ function MoveData.calcHiddenPowerTypeAndPower(ivs)
 	return moveType, movePower
 end
 
+---Adjusts the move table data based on any variable damage calculations, or other attributes; No return, as this edits the move table directly.
+---@param move table
+---@param sourcePokemon? table Optional, as not all move adjustment calculations require a source and/or a target
+---@param targetPokemon? table Optional, as not all move adjustment calculations require a source and/or a target
+function MoveData.adjustForVariablePower(move, sourcePokemon, targetPokemon)
```

Comment:
Same as above, these don't just adjust move power so this should be more generic e.g. `adjustVariableMoveValues` or something